### PR TITLE
[WIP] emodnet multifile netcdf bathymetry reader

### PR DESCRIPTION
**Pull request by poplarShift**
I couldn't find .dtm files on the EMODNET download portal, so rewrote the TopoReader with netcdf files.

While at it, also enabled loading of multi-file datasets to enable grids spanning tiles (and not having to search for the right tile).

I assume this will need to be polished but it's a good starting point I believe

### CODE REVIEW

**Comment by bjorkqvi** [comment]
Thanks for this. Looks like the "normal" EMODNET reader is just a special case of your reader, and it would therefore make sense to just replace the original reader with your reader and not have the user think about it. But I will fetch it, test it, and merge it into the dev-branch. We have some other minor improvements, so we can then include everything in the next release in the main branch.

**Comment by poplarShift** [comment]
Looks like this PR does not work for xarray<0.19 which is what you require.

Works like a charm with current xarray 0.20.1, but then e.g. this: https://github.com/KonstantinChri/dnora/blob/23c0218481c912af6c29dfb1b1c1a802180cc9ac/dnora/wnd/read_metno.py#L121 has to be updated I think.

We'll also have to require dask for open_mfdataset to work.

**Comment by bjorkqvi** [comment]
I know @KonstantinChri worked with the xarray version issue. Maybe he can comment on how to solve this? If we can make it work with the current version then that is of course preferable, since we don't want legacy issues.

**Comment by KonstantinChri** [comment]
I updated the enviroment.yml, so we can use a new version of xarray (>=0.20) and I updated the lines 121-122 in read_metno.py as @poplarShift suggested. Now it seems to work ok.

**Comment by bjorkqvi** [comment]
I finally got to this, but I can't find any netcdf (only DTM) download in the emodnet portal, so I can't test the code. Can you provide a link to where I can get the files needed to do the test?

File: dnora/grd/read.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def _crop(ds):
             return ds.isel(lon=slice(2, -1), lat=slice(2, -1))
 
         import dask
-        with dask.config.set(**{'array.slicing.split_large_chunks': True}):
```
Comment (poplarShift) [inline dnora/grd/read.py]: also I now think that should be True here. Haven't properly benchmarked it though.

Comment (bjorkqvi) [reply dnora/grd/read.py]: Ok, I changed that flag to True.

**Pull request merged**